server/app/services/export/CsvExporter.java

```diff
@@ -21,60 +21,40 @@
  * CsvExporter takes a list of {@link Column}s and exports the data specified. A column contains a
  * {@link Path} indexing into an applicant's data, and CsvExporter takes the path and reads the
  * answer from {@link ReadOnlyApplicantProgramService} if present.
+ *
+ * <p>The close method should be called when complete in order for the underlying {@link CsvPrinter}
```

Comment:
According to the java docs, close is called automatically if you have a try-with-resources.  This comment seems a bit incorrect.  Could you modify it to say you need to either close() or  try-with-resources?